packages/meditrak-server/src/log.ts

```diff
@@ -0,0 +1,62 @@
+/**
+ * Tupaia
+ * Copyright (c) 2017 - 2020 Beyond Essential Systems Pty Ltd
+ */
+
+// TODO: Delete this file in favour of configureWinston() from server-boilerplate
```

Comment:
This file can be deleted once [this refactor PR](https://github.com/beyondessential/tupaia/pull/3900) is approved and merged into dev